Ch03-Simple-sampling-Monte-Carlo-methods/problem-1-2-3/solution.jl

```diff
@@ -0,0 +1,66 @@
+using Random: default_rng
+using Random
+using Distributions
+
+_checkN(n) = n ≥ 1 || throw(ArgumentError("number of samples ($N) must be ≥ 1"))
+
+"""
+    hit_miss(f, [rng=GLOBAL_RNG], (lbx, ubx), (lby, uby), N::Integer)
+
+integrate function `f` with the 'hit or miss' method, sampling from the rectangle
```

Comment:
Maybe: importance sampling method between ‘lbx’ and ‘ubx’ is used to integrate function f